defs/scenarios/storage/ceph/ceph-mon/unresponsive_mon_mgr.yaml

```diff
@@ -0,0 +1,45 @@
+checks:
+  # In older sosreports, the plugin is called 'ceph' (= mon + mgr + osd) and
+  # in the newer ones it's called 'ceph_mon'.
+  sosreport_hung_ceph_mon_old:
+    requires:
+      property:
+        path: hotsos.core.plugins.sosreport.SOSReportChecksBase.timed_out_plugins
+        ops: [[eq, ceph]]
```

Comment:
timed_out_plugins is a list of plugins so you will need to do:

ops: [[contains, ceph]]

otherwise this will never match. same goes for the other checks.

Reply:
done